### PR TITLE
links to forum and related updates

### PR DESCRIPTION
**Pull request by jseldess**
Covers some of #463

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/518)

### CODE REVIEW

File: contribute-to-cockroachdb.md [hunk 1]
```diff
@@ -6,20 +6,18 @@ toc: false
...
\ No newline at end of file
+4. Make your changes and follow our [code review workflow](https://github.com/cockroachdb/cockroach/blob/master/CONTRIBUTING.md#code-review-workflow).
+
+## Report a Bug
+
+For filing bugs, suggesting improvements, or requesting new features, help us out by [opening an issue](https://github.com/cockroachdb/cockroach/issues/new).
+
+## Need Help?
+
+- [CockroachDB Forum](https://forum.cockroachlabs.com): Ask questions, find answers, and help other users.
```
Comment (jess-edwards) [inline contribute-to-cockroachdb.md]: CockroachDB Community Forum

File: frequently-asked-questions.md [hunk 2]
```diff
@@ -149,5 +149,5 @@ CockroachDB is a distributed SQL database built on a transactional and strongly-
 
 ## Have questions that weren’t answered? 
 
-- [Join us on Gitter](https://gitter.im/cockroachdb/cockroach). This is the best, most immedate way to connect with CockroachDB engineers. 
-- [Post to our Developer mailing list](https://groups.google.com/forum/#!forum/cockroach-db). Please join first or your messages may be held back for moderation.
+- [CockroachDB Forum](https://forum.cockroachlabs.com): Ask questions, find answers, and help other users.
```
Comment (jess-edwards) [inline frequently-asked-questions.md]: CockroachDB Community Forum

**Comment by jess-edwards** [comment]
@jseldess Minor comments inline. Otherwise LGTM.

**Comment by jseldess** [comment]
Thanks, Jess.

---

Review status: 0 of 3 files reviewed at latest revision, 3 unresolved discussions.

---

*[contribute-to-cockroachdb.md, line 22 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/518#-KNjPdZKjVUauRl0LJCw-r1-22:-KNm04xIfiXbPkqormMl:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/2bfc6a44a8b79d49dda54a9411ce1520192af9ed/contribute-to-cockroachdb.md#L22)):*
<details><summary><i>Previously, jess-edwards wrote…</i></summary>
> CockroachDB Community Forum
</details>
Done.

---

*[frequently-asked-questions.md, line 152 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/518#-KNjPdZKjVUauRl0LJCx-r1-152:-KNm05Ved3xmylJSfNit:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/2bfc6a44a8b79d49dda54a9411ce1520192af9ed/frequently-asked-questions.md#L152)):*
<details><summary><i>Previously, jess-edwards wrote…</i></summary>
> CockroachDB Community Forum
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/518#-:-KNm066L5f86NeLaXZrr:1433396657)*

**Pull request merged**